src/hermes_claude_agent_sdk/parity/v4_live_scenarios.py

```
@@ -0,0 +1,318 @@
+        raise V4LiveScenarioViolation("scenario catalog requires the frozen corrected v4 live map")
+    return document, source, accounting
+
+
+def _from_map(document: Mapping[str, Any], accounting: Mapping[str, Any]) -> V4LiveScenarioCatalog:
+    children: dict[str, list[Mapping[str, Any]]] = {}
+    for child in document["child_calls"]:
+        children.setdefault(child["row_key"], []).append(child)
+    scenarios = []
+    for row in document["rows"]:
+        key = f"{row['source_pack']}/{row['source_item_id']}"
+        trials = tuple(row["required_trial_indexes"]); parent_calls = row["parent_calls"]
```

Ruff (0.16.3)

[error] Multiple statements on one line (semicolon)

(E702)